modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/ManagementToolbar/ManagementToolbarColumns.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import {useContext, useState} from 'react';
+
+import {ListViewContext, ListViewTypes} from '../../context/ListViewContext';
+import i18n from '../../i18n';
+import Form from '../Form';
+import {Column} from '../Table';
+
+type ManagementToolbarColumnsProps = {
+	columns: Column[];
+	onClose: () => void;
+};
+
+type CollumnsState = {
+	[key: string]: boolean;
+};
+
+const ManagementToolbarColumns: React.FC<ManagementToolbarColumnsProps> = ({
+	columns,
+	onClose,
+}) => {
+	const [{columns: contextColumns, columnsFixed}, dispatch] = useContext(
+		ListViewContext
+	);
+
+	const columnsNotFixed = columns.filter(
+		({key}) => !columnsFixed.includes(key)
+	);
+
+	const [selectedColumns, setSelectedColumns] = useState<CollumnsState>(
+		() => {
+			const newColumns: CollumnsState = {};
+
+			columnsNotFixed.forEach(({key}) => {
+				newColumns[key] = contextColumns[key] ?? true;
+			});
+
+			return newColumns;
+		}
+	);
+
+	const disabled = Object.values(selectedColumns).every(
+		(visible) => !visible
+	);
+
+	return (
+		<div className="align-content-between d-flex flex-column">
+			<div className="dropdown-header">
+				<p className="font-weight-bold my-2">
+					{i18n.translate('columns')}
+				</p>
+			</div>
+
+			<div className="body-toolbar">
```

Comment:
I think the name `body-toolbar` is too vague.

Put a classname that emphasizes the behavior

```suggestion
			<div className="body-toolbar">
```